astroid/nodes/scoped_nodes.py

```diff
@@ -504,6 +505,8 @@ def __init__(
         :param parent: The parent node in the syntax tree.
         :type parent: NodeNG or None
 
+        :param doc_node: The doc node associated with this node.
```

Comment:
Could you also add the updated docstring to all these docstrings?

Reply:
Updated was what?

Reply:
The updated docstring has become outdated 😄